src/auth/computeclient.ts

```diff
@@ -68,8 +71,11 @@ export class Compute extends OAuth2Client {
     try {
       // TODO: In 2.0, we should remove the ability to configure the tokenUrl,
       // and switch this over to use the gcp-metadata package instead.
-      res = await this.transporter.request<CredentialRequest>(
-          {url, headers: {'Metadata-Flavor': 'Google'}});
+      res = await ax.request<CredentialRequest>({
+        url,
+        headers: {'Metadata-Flavor': 'Google'},
```

Comment:
You have a typo in those headers, and without an end-to-end deployment test we wouldn't have caught this.  We will only find out when something is actually deployed by a user.

Actually there is no typo (I hope), but I had you for a second.

Can you use an exported value of this constant from `gcp-metadata` rather than typing it again, potentially with typos.

Reply:
I fixed this to use the existing HEADER_NAME, and submitted a PR to gcp-metadata to export the whole header 👍 
https://github.com/stephenplusplus/gcp-metadata/pull/33